erpnext/stock/module_onboarding/stock/stock.json

```diff
@@ -31,15 +31,15 @@
   {
    "step": "Create a Product"
   },
-  {
-   "step": "Create a Supplier"
-  },
   {
    "step": "Introduction to Stock Entry"
   },
   {
    "step": "Create a Stock Entry"
   },
+  {
+   "step": "Create a Supplier"
```

Comment:
Let's keep the basic setups first so supplier aftrt product is okay, btw supplier was way down earlier. I got feedback to move it up :)